packages/cascara/src/ui/ActionsMenu/ActionsMenu.js

```diff
@@ -1,23 +1,18 @@
 import pt from 'prop-types';
-import React, { useLayoutEffect, useRef } from 'react';
-import { Button } from 'reakit/Button';
+import React, { useContext, useLayoutEffect, useRef } from 'react';
 import { Menu, MenuButton, MenuItem, useMenuState } from 'reakit/Menu';
-import styles from './ActionsMenu.module.scss';
 
+import styles from './ActionsMenu.module.scss';
 import { popperOverTrigger } from '../../shared/popperModifiers';
-
-const DEFAULT_TRIGGER = (
-  <Button className='ui basic icon button'>
-    <b>⋯</b>
-  </Button>
-);
-// const DEFAULT_TRIGGER = <p>'Trigger'</p>;
```

Comment:
@lordpixel we don't want to lose this. We will probably never need to define a different trigger for the ActionsMenu to open than this. But if we do not define this, people will be guessing what was used each time.

The way I was able to see this... if you look at the button in your screenshots, the ellipsis icon is now at the bottom of the button instead of the middle like this character.

Let's bring this back so we do not need to define a trigger when using ActionsMenu.

Reply:
working on it